### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by blcts**

### CODE REVIEW

File: src/robotFactory.js [hunk 1]
```diff
@@ -1,10 +1,79 @@
...
+  }
+}
+
+class FlyingRobot extends BaseRobot {
+  constructor(name, weight, coords, chipVersion) {
+    super(name, weight, coords, chipVersion);
+
+    this.coords = {
+      x: coords.x || 0,
+      y: coords.y || 0,
```
Comment (nikita-biriukov) [inline src/robotFactory.js:44]: you can not copy x and y coords
you can just add z coord

**Review by nikita-biriukov — APPROVED** [review]
good job 🔥 🔥

File: src/robotFactory.js (hunk 1 above)
Comment (solaryasha) [reply src/robotFactory.js:44]: Agree with Nikita, `super` method already done it.
So, remove these lines

**Review by solaryasha — CHANGES_REQUESTED** [review]

**Review by VETALEK — APPROVED** [review]
🚀

**Pull request closed without merging**